src/clojure_by_example/ex03_data_and_functions.clj

```diff
@@ -154,24 +154,6 @@
 #_('FIX 'FIX 'FIX)
 
 
-
-;; Lesson: Thinking with Data:
```

Comment:
I think this ought to be removed because:
1. It's quite a dense paragraph, requiring a careful reading to understand properly.
2. It doesn't really have anything to do with learning Clojure.

It's really just general programming advice which I feel is distracting at this stage of the workshop, where a participant is still understanding how `map` and `filter` work.

Reply:
Hm, it is. OK, we can answer if someone asks "but why?" Will merge this in.